### PR TITLE
make TX fee as a reward

### PR DESCRIPTION
**Pull request by 11Yupi11**
### Summary
Created new state (AccFee) to collect all parts of transaction fees and simple function to send them to this state, to get the ammount and to restart from 0.

### Purpose of change
Before the change all transaction fees were burned. Now we propose to burn only part of them and other part save in future to distribute between validators as an additional reward for staying online and  the developers team.

### Solution
Added State Object with AccFee to collect on coinbase address all information about the amount of needed small parts. At the place where all transaction fee were burned add some calculations to count the amount that need to be saved. Also function to add this ammount to the state object added.
To make possibility to get the information about state and update it to 0 value functions GetAccFee and Set0AccFee were added.

### Issue
Some unit tests are broken

### CODE REVIEW

**Comment by DerTiedemann** [comment]
Looks good so far, not sure if we need a sunrise phase here, e.g. we could use the already existing SunrisePhase0 or just leave it as it is. But as a thing that would be needed is to add the accululation address to the genesis block and read it from there.

**Comment by 11Yupi11** [comment]
I have added all logic to take and work with Accumulative address from genesis file

File: core/state_transition.go [hunk 1]
```diff
@@ -350,7 +350,13 @@ func (st *StateTransition) TransitionDb() (*ExecutionResult, error) {
 		ret, st.gas, vmerr = st.evm.Call(sender, st.to(), st.data, st.gas, st.value)
 	}
 	st.refundGas(apricotPhase1)
-	st.state.AddBalance(st.evm.Context.Coinbase, new(big.Int).Mul(new(big.Int).SetUint64(st.gasUsed()), st.gasPrice))
+
+	totalFee := new(big.Int).Mul(new(big.Int).SetUint64(st.gasUsed()), st.gasPrice)
+	onePercentCalc := new(big.Int).Div(totalFee, new(big.Int).SetUint64(100))
+	exportedPartFee := new(big.Int).Mul(onePercentCalc, new(big.Int).SetUint64(60))
```
Comment (evlekht) [inline core/state_transition.go]: You should multiply things first before you divide them in such big.Int calculations - that way you will lose less precision.

**Review by evlekht — CHANGES_REQUESTED** [review]

**Comment by peak3d** [comment]
Replaced by #15 and ongoing development of @pnowosie

**Pull request closed without merging**